src/btree/bt_delete.c

```diff
@@ -376,6 +378,7 @@ __wt_delete_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
 
             if (page_del != NULL)
                 page_del->update_list[count++] = upd;
+            total_size += size;
```

Comment:
my preference is size increment statement immediately after the memory allocation, for eg:- in this case move line 381 above line 376.

Reply:
Moved the increment to the allocation location.